cloudwatch_metrics_collector/src/metrics_collector_parameter_helper.cpp

```diff
@@ -161,16 +163,42 @@ void ReadStorageResolution(
 
 void ReadTopics(
         std::shared_ptr<Aws::Client::ParameterReaderInterface> parameter_reader,
-        std::vector<std::string> & topics) {
+        std::vector<TopicInfo> & topics) {
```

Comment:
(probably to be tackled in a different PR) It is strange to have `ReadTopics` returning void and having an out parameter (`topics`). Do we expect `topics` to be non-empty when we start?

Reply:
Yes, that would be a different PR that will change more than just this function.